docs/.vitepress/config.js

```diff
@@ -57,7 +57,7 @@ function getMenuSidebar() {
         { text: 'Soul', link: '/playlist/soul' },
         {text: 'Soundtrack', link: '/playlist/soundtrack'},
         { text: 'Trap', link: '/playlist/trap' },
-        { text: 'Hip-hp', link: '/playlist/hip-hop' }
+        { text: 'Hip-hop', link: '/playlist/hip-hop' }
```

Comment:
Please sort alphabetically for add new genre inside that function, refer to this https://awesome-music-playlist.vercel.app/contribution.html#add-new-genre